OpenRA.Game/Network/OrderManager.cs

```diff
@@ -165,6 +168,22 @@ void CheckSync(byte[] packet)
 				syncForFrame.Add(frame, packet);
 		}
 
+		Dictionary<int, ulong> winStateForFrame = new Dictionary<int, ulong>();
+
+		void CheckWinState(byte[] packet)
```

Comment:
This must be done on the server and not on the client.

Reply:
Done.